sites/public/pages/listings.tsx

```diff
@@ -65,7 +65,9 @@ export async function getStaticProps() {
   let closedListings = []
 
   try {
-    const response = await axios.get(process.env.listingServiceUrl)
+    const response = await axios.get(process.env.listingServiceUrl, {
+      params: { page: "1", limit: "10" },
+    })
```

Comment:
Is this bit necessary for the hard coded pagination? Asking because I'm messing around in this file too and it'd be great (if optimistic) if we can avoid merge conflicts.

Reply:
If I don't hard code it here, I'd want to redirect this page to `/listings/page/1`, because all the links currently point to `/listings` and that would attempt to load all 300 properties. You can overwrite this with whatever your version has.

Reply:
Ah sorry, I was totally missing the point of this PR. This is fine for now so the main page won't show all the listings. The conflicts should be easy to fix.